mesh/accounts/models.py

```diff
@@ -3,8 +3,11 @@
 class Account(models.Model):
     accountID = models.AutoField(primary_key=True)
     email = models.EmailField()
-    encryptedPass = models.CharField(max_length=64)
-    
+
+    #please set editable to False after debugging process
+    encryptedPass = models.BinaryField(max_length=64,editable = True)
+    salt = models.BinaryField(max_length = 36,editable = True) 
+    ###
```

Comment:
What's this for?

Reply:
editable is to be used for debugging in the admin console. I changed encryptedPass to a BinaryField to make storing and retrieving easier. Salt is stored alongside the password, but because I'm using Bcrypt, I don't really know any delimiters they don't use, so I figured it's more efficient to store it as a new field.